assignment0_bibek/MM_MV_Product.cpp

```diff
@@ -0,0 +1,45 @@
+#include "MM_MV_Product.h"
+#include <iostream>
+#include <vector>
+
+// Function to multiply two matrices
+std::vector<std::vector<double>> M_M_Product(const std::vector<std::vector<double>>& matrixA, const std::vector<std::vector<double>>& matrixB){
```

Comment:
Consider using an alias type to avoid writing `vector<vector<double>>` everywhere. i.e., `using Matrix = std::vector<std::vector<double>>`

 It is fine for this assignment, however `vector<vector>>` will not tend to give good performance since each column is an independent allocation.